### ISSUE TITLE
Release v1.59.8

### ISSUE THREAD

**Issue by deckhouse-BOaTswain** [deckhouse/deckhouse#8250]
:robot: A dedicated issue to run tests and deploy release [v1.59.8](https://github.com/deckhouse/deckhouse/milestone/307).

---

<details>
<summary>Release issue commands and options</summary>
<br />

You can trigger release related actions by commenting on this issue:

- `/deploy/<channel>[/<editions>]` will publish built images into the release channel.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
  - `editions` is a comma-separated list of editions `ce | ee | fe | be | se`
- `/suspend/<channel>` will suspend released version.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
- `/e2e/run/<provider> git_ref` will run e2e using provider and an `install` image built from git_ref.
  - `provider` is one of `aws | azure | eks | gcp | openstack | static | vsphere | yandex-cloud`
  - `git_ref` is a tag `v1.59.8 | test-v1.59.8*` or a branch `main | release-1.59`
- `/e2e/run/<provider> git_ref_1 git_ref_2` will run e2e using provider and git_ref_1 and then switch to git_ref_2.
  - `provider` is one of `aws | azure | eks | gcp | openstack | static | vsphere | yandex-cloud`
  - `git_ref_1` is a release-* or main branch
  - `git_ref_2` is a release-* or main branch
- `/e2e/use/k8s/<version>` specifies which Kubernetes version to use for e2e test.
  - `version` is one of `1.25 | 1.26 | 1.27 | 1.28 | 1.29`
- `/build git_ref` will run build for release related refs.
  - `git_ref` is a tag `v1.59.8 | test-v1.59.8*` or a branch `main | release-1.59`

**Note 1:**
A single command `/e2e/run/<provider>` will run e2e with default CRI 'containerd' and Kubernetes version '1.25'.
Put `/e2e/use` options below `/e2e/run` command to set specific CRI and Kubernetes version. E.g.:

```
/e2e/run/aws main
/e2e/use/k8s/1.27
/e2e/use/k8s/1.29

This comment will run 2 e2e jobs on AWS with containerd
and with Kubernetes version 1.20 and 1.23 using image built from main branch.
```

```
/e2e/run/aws release-1.35 release-1.36
/e2e/use/k8s/1.27

This comment will create cluster in AWS using Deckhouse built from release-1.35 branch
and then switch to images built from release-1.36 branch.
'use' options are supported, cluster will use containerd and Kubernetes version 1.23.
```

**Note 2:**
'deploy', 'suspend' and 'e2e' commands should run after 'build FE' job is finished.

**Note 3:**
No autobuild for release branch. Run this command after cherry-picking into release branch:
```
/build release-1.59
```

</details>

**Comment by deckhouse-BOaTswain** [comment]
New commit [bd510d](https://github.com/deckhouse/deckhouse/commit/bd510df8970b110ef6f466749e4d0496f4bc1acc) in branch **main**:
```
Merge pull request #8248 from deckhouse/changelog/v1.59.7

Changelog v1.59.7
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [03dd69](https://github.com/deckhouse/deckhouse/commit/03dd693b277d62962cdecbaff53a42fb48d03451) in branch **main**:
```
[monitoring-kubernetes] Avoid error on release decode (#8251)

Signed-off-by: Yuriy Losev <yuriy.losev@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [6b4efe](https://github.com/deckhouse/deckhouse/commit/6b4efeb9d235a4e12252fc1526c3efee28590a12) in branch **main**:
```
Merge pull request #8253 from deckhouse/changelog/v1.59.7

Changelog v1.59.7
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [4c3562](https://github.com/deckhouse/deckhouse/commit/4c3562f71e7d110567459e17dc092d053be1256b) in branch **main**:
```
[deckhouse] replace go-based conversions with jq-based (#8193)

Signed-off-by: Stepan Paksashvili <stepan.paksashvili@flant.com>
Signed-off-by: Yuriy Losev <yuriy.losev@flant.com>
Co-authored-by: Stepan Paksashvili <stepan.paksashvili@flant.com>
Co-authored-by: Yuriy Losev <yuriy.losev@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [904ce0](https://github.com/deckhouse/deckhouse/commit/904ce03418d769818786f5d51c17b4f3e8281000) in branch **main**:
```
[candi] set minimum supported version debian 10 (#7710)

Signed-off-by: Nikolay Gorbatov <nikolay.gorbatov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [9776a1](https://github.com/deckhouse/deckhouse/commit/9776a13333b979d24be359857e4fb602e7b27e9b) in branch **main**:
```
[chore] Update K8s patch version (#8239)

Signed-off-by: Artem Kladov <artem.kladov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [6b2f2f](https://github.com/deckhouse/deckhouse/commit/6b2f2fd58be0fd627629e8be55e150a3a075134a) in branch **main**:
```
[cloud-provider-yandex] add custom target group node annotation (#8171)

Signed-off-by: Denis Romanenko <denis.romanenko@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [feb71f](https://github.com/deckhouse/deckhouse/commit/feb71fe9ad97bdeaf633fefbdbad554c4cbb55f1) in branch **main**:
```
[node-manager] bump mcm version (#8194)

Signed-off-by: Mikhail Scherba <mikhail.scherba@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [757ffd](https://github.com/deckhouse/deckhouse/commit/757ffd34012927d2490f8f9cc830e098029ff904) in branch **main**:
```
[dhctl] embed version into binary (#8231)

Signed-off-by: Maxim Vasilenko <maksim.vasilenko@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [5d1de9](https://github.com/deckhouse/deckhouse/commit/5d1de9d5dbfbf321e217d145e3a9472d893c3a91) in branch **main**:
```
[testing] fix test fluctation (#8257)

Signed-off-by: Nikolay Gorbatov <nikolay.gorbatov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [a30998](https://github.com/deckhouse/deckhouse/commit/a309981f56586320dbd9c4c04bb05e069adcab03) in branch **main**:
```
[deckhouse] Validate that registry address is always present (#8242)

Signed-off-by: Maxim Vasilenko <maksim.vasilenko@flant.com>
Co-authored-by: Maxim Vasilenko <maksim.vasilenko@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [ebb44f](https://github.com/deckhouse/deckhouse/commit/ebb44ffb33a6206fc2276a085cf4357fb2251bca) in branch **main**:
```
[l2-load-balancer] Documentation fixes + declaring module as experimental (#8236)



Signed-off-by: Andrey Polovov <andrey.polovov@flant.com>

Co-authored-by: Konstantin Aksenov <konstantin.aksenov@flant.com>
Co-authored-by: Konstantin Nezhbert <zhbert@yandex.ru>
Co-authored-by: Kirill Kononovich <41591254+kirkonru@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New tag **v1.59.8** is created.
